### PR TITLE
feat(EMS-3767-3768-3781): business - turnover currency page and check your answers

### PR DESCRIPTION
**Pull request by Zainzzkk**
## Introduction :pencil2:
This PR moves the turnover alternative currency page to a standalone page and check your answers

## Resolution :heavy_check_mark:
* Made alternative currency for turnover a standalone page
* Removed link from turnover page
* Updated tests
* Updated and added e2e tests
* Added new row for turnover currency
* Added change your answers for turnover currency

## Miscellaneous :heavy_plus_sign:
* Added temporary clickAlternativeCurrencyLink while other alternative currency pages are being moved

### CODE REVIEW

File: e2e-tests/commands/insurance/complete-and-submit-alternative-currency-form.js [hunk 1]
```diff
@@ -11,19 +11,27 @@ const { CURRENCY_CODE } = INSURANCE_FIELD_IDS.CURRENCY;
...
+    }
 
     radios(CURRENCY_CODE, isoCode).option.label().click();
     cy.clickSubmitButton();
   }
 
   if (alternativeCurrency) {
-    cy.clickProvideAlternativeCurrencyLink();
+    if (clickAlternativeCurrencyLink) {
+      cy.clickProvideAlternativeCurrencyLink();
```
Comment (ttbarnes) [inline e2e-tests/commands/insurance/complete-and-submit-alternative-currency-form.js:26]: Are all usages of this temporary until the new form is fully working?

File: e2e-tests/commands/insurance/your-business/complete-and-submit-your-business-forms.js [hunk 2]
```diff
@@ -5,11 +5,12 @@
  * @param {String} formToStopAt: the form to stop at
  * @param {Boolean} hasCreditControlProcess: whether the exporter has a credit control process
  * @param {Boolean} differentTradingAddress: whether the exporter has a different trading address
+ * @param {Boolean} differentTradingAddress: whether the exporter has a different trading name
```
Comment (ttbarnes) [inline e2e-tests/commands/insurance/your-business/complete-and-submit-your-business-forms.js:8]: typo

File: e2e-tests/commands/insurance/complete-and-submit-alternative-currency-form.js (hunk 1 above)
Comment (Zainzzkk) [reply e2e-tests/commands/insurance/complete-and-submit-alternative-currency-form.js:26]: yes, once all the pages have the alternate currency removed,  then I can rework this slightly.  clickAlternativeCurrencyLink will be removed

File: src/ui/server/controllers/insurance/business/turnover/index.ts [hunk 3]
```diff
@@ -55,27 +48,9 @@ const { TURNOVER: TURNOVER_FIELDS } = FIELDS;
...
-    alternativeCurrencyUrl = `${INSURANCE_ROOT}/${referenceNumber}${TURNOVER_ALTERNATIVE_CURRENCY_CHANGE}`;
-  }
-
-  /**
-   * If checkAndChangeRoute,
-   * URL should be ALTERNATIVE_CURRENCY_CHECK_AND_CHANGE
-   */
-  if (checkAndChangeRoute) {
-    alternativeCurrencyUrl = `${INSURANCE_ROOT}/${referenceNumber}${TURNOVER_ALTERNATIVE_CURRENCY_CHECK_AND_CHANGE}`;
-  }
```
Comment (ttbarnes) [inline src/ui/server/controllers/insurance/business/turnover/index.ts:77]: So nice to be able to remove this now 🎉

File: src/ui/templates/insurance/your-business/turnover.njk [hunk 4]
```diff
@@ -75,10 +75,6 @@
             validationError: validationErrors.errorList[annualTurnoverId]
           }) }}
 
-          <p class-"govuk-body">
-            <a class="govuk-link" data-cy="provide-alternative-currency-link" href="{{ PROVIDE_ALTERNATIVE_CURRENCY_URL }}">{{ CONTENT_STRINGS.PROVIDE_ALTERNATIVE_CURRENCY }}</a>
```
Comment (ttbarnes) [inline src/ui/templates/insurance/your-business/turnover.njk:79]: Note - after other pages have the same treatment ("buyer - outsanding payments", "export contract - agent charges"), we can remove all instances of `provide-alternative-currency-link`.

**Review by ttbarnes — APPROVED** [review]

**Pull request merged**